src/modules/facility-activation-transaction/dto/create-facility-activation-transaction-request.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { ENUMS } from '@ukef/constants';
+import { ValidatedFacilityIdentifierApiProperty } from '@ukef/decorators/validated-facility-identifier-api-property';
+import { ValidatedNumberApiProperty } from '@ukef/decorators/validated-number-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { UkefId } from '@ukef/helpers';
+
+export type CreateFacilityActivationTransactionRequest = CreateFacilityActivationTransactionRequestItem[];
+
+export class CreateFacilityActivationTransactionRequestItem {
+  @ValidatedFacilityIdentifierApiProperty({
+    description: 'The identifier of the facility to activate.',
+  })
+  readonly facilityIdentifier: UkefId;
+
+  @ValidatedNumberApiProperty({
+    description: 'In most situations value should be 3, it means auto approval.',
+    example: ENUMS.BUNDLE_STATUSES.SUBMIT_FOR_POSTING,
+    minimum: 0,
```

Comment:
Do we need `length` decorator?

Reply:
Don't think that we need it for numbers.
In this case we limit input to values in Enum. This reduces possible input values to 3.